src/main/java/org/frc5687/infiniterecharge/robot/subsystems/Hood.java

```diff
@@ -8,20 +8,26 @@
 import org.frc5687.infiniterecharge.robot.Constants;
 import org.frc5687.infiniterecharge.robot.OI;
 import org.frc5687.infiniterecharge.robot.RobotMap;
+import org.frc5687.infiniterecharge.robot.util.Limelight;
 import org.frc5687.infiniterecharge.robot.util.OutliersContainer;
 
 public class Hood extends OutliersSubsystem {
 
     private OI _oi;
     private TalonSRX _hoodController;
+    private Limelight _limelight;
 
     private double _positionABS;
     private double _position;
     private double _setPoint;
 
-    public Hood(OutliersContainer container, OI oi) {
+    private Limelight.Pipeline _pipeline = Limelight.Pipeline.Wide;
+
+    public Hood(OutliersContainer container, Limelight limelight, OI oi) {
         super(container);
+        _limelight = limelight;
         _oi = oi;
+        _limelight = limelight;
```

Comment:
we probably don't need to initialize limelight twice here? :) 

[if we do need to initialize it twice then oi might be doing something super wrong]

Reply:
You are correct, has been fixed